websites/B/BloomHost/presence.ts

```diff
@@ -0,0 +1,111 @@
+const presence = new Presence({
+		clientId: "873728851052752896"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ hostname, pathname } = document.location;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "bloom_logo",
+		startTimestamp: browsingTimestamp
+	};
+	if (hostname === "bloom.host") {
+		presenceData.smallImageKey = "globe-solid";
+		presenceData.details = "Viewing the main website";
+		if (pathname.includes("minecraft")) {
+			presenceData.details = "Viewing the main website";
```

Comment:
no need to set this again, it was set two lines above, same for all the ones below this